docs/developing/information/indexers.md

```diff
@@ -1,429 +1,186 @@
 
-The highlighted part at the top of the screenshot is the command we used:
-
-`tezos-client rpc get /chains/main/blocks/2283698/operations/3/2`
-
-The numbers in the command mean the following:
-
-- 2283698 — the level of the block in which the operation took place.
-- 3 — the index of the operation's source type. "3" means the user initiated it.
-- 2 — the index of the operation in a particular block.
-
-To get the details of a specific operation, we need to know its location in the blockchain: when it was included, what operation type table it was written in, and its number in that index. To know it, we need to check all blocks in turn for the presence of the specified hash, which is very slow and ineffective.
-
-Indexers simplify the task of getting the data you need from the blockchain. They request whole blocks from public nodes and write them to their databases. Then indexers create indexes—-additional data structures optimized for fast data retrieval that store either the data or a link to the corresponding place in the central database. When searching for any data, the indexer looks for it in the corresponding index, not in the central database. For even faster searching, some indexers don't store all data in one table but create separate tables for any valuable data.
-
-![](/img/usdt_que_pasa.png)
+<!-- In addition to search speed, indexing has another advantage: the ability to modify indexing rules. For example, TzKT provides an additional index, where each Tezos FA1.2 and FA2 token has its internal id. So instead of comparing relatively long contract addresses, it will compare small numbers and retrieve data even faster. -->
 
-Here is a USDT contract indexed with Que Pasa. It made tables for every entry point and big_map in the contract and an index for every table. It is an efficient way to fetch data. For example, the execution of a query to find a balance of a random USDT holder took only 0.064 milliseconds.
+<!-- TODO what does this mean? Is it the convenience of having `id=5` instead of having to remember the address of the contract? Here's my rewrite: -->
+<!-- This token now appears to have the ID 42290944933889; is that really better than the contract address? The example used id=85. -->
 
-![](/img/que_pasa_sql.png)
+Indexers can organize data in custom ways.
+For example, TzKT indexes FA1.2 and FA2 tokens and gives each an internal ID.
+This ID can be faster and easier to work with and compare than the contract ID.
 
-The exact list of tables, indexes schemas, and command syntax depends on the indexer and database it uses. While using TzKT, a query for a USDT balance will look like this:
+This TzKT query gets information about an FA1.2 token based on its internal ID instead of its contract address:
 
 ```
-https://api.tzkt.io/v1/tokens/balances?token.contract=KT1XnTn74bUtxHfDtBmm2bGZAQfhPbvKWR8o&account=tz1a1RTsGUbads3VucUQDxJF4EDXkDWcDHPK
+https://staging.api.tzkt.io/v1/tokens?id=42290944933889
```

Comment:
https://api.tzkt.io/v1/tokens?id=42290944933889 is better

Reply:
Updated.